src/VisualStudio/VisualBasic/Impl/BasicVSResources.resx

```diff
@@ -307,4 +307,7 @@
   <data name="Insert_apostrophe_at_the_start_of_new_lines_when_writing_apostrophe_comments" xml:space="preserve">
     <value>Insert ' at the start of new lines when writing ' comments</value>
   </data>
+  <data name="Show_hints_for_New_expressions" xml:space="preserve">
+    <value>Show hints for 'New' expressions</value>
```

Comment:
Not shared because of new/New casing? (fine, just wanted to confirm.)

Reply:
correct.  don't love it, but it seems the most proper